include/networkit/distance/DynBFS.hpp

```diff
@@ -43,6 +47,7 @@ class DynBFS final : public DynSSSP {
     enum Color { WHITE, BLACK, GRAY };
     std::vector<Color> color;
     count maxDistance;
+    const count infDist = std::numeric_limits<count>::max();
```

Comment:
This can be a static constexpr

Reply:
Are we sure of it?

the regular version of BFS had it as just `const`. Upon trying to make it work as `static constexpr` I have only been getting linker errors.

Reply:
Yes, one example is in [BidirectionalDijkstra](https://github.com/networkit/networkit/blob/master/include/networkit/distance/BidirectionalDijkstra.hpp#L51)